Snapper/Manager.py

```diff
@@ -0,0 +1,74 @@
+from Snapper.Snapper import Snapper
+from Snapper.AssociationMatrix import AssociationMatrix
+from Relationship.Variable import Variable
+from Relationship.SpearframeRelationship import SpearframeRelationship
+
+
+class Manager:
+
+    def __init__(self):
+        self.sensors = []
+        self.route_map = {}
+        self.variables = []
+        self.snapper = Snapper(self)
+        self.matrix = AssociationMatrix()
+
+    def add_sensor(self, sensor):
```

Comment:
That is what I understand and what I will be doing tonight unless I hear differently